docs/content/developer/getting-started/install-iota.mdx

```diff
@@ -73,18 +73,24 @@ Once you have downloaded and extracted the `.tgz` file that matches your operati
 and
 install the necessary binaries. You should start with the main IOTA binary.
 
-1. iota-`<OS>`-`<ARCHITECTURE>`: Main IOTA binary.
-2. iota-faucet-`<OS>`-`<ARCHITECTURE>`: Local faucet to mint coins on a local network.
-3. iota-indexer-`<OS>`-`<ARCHITECTURE>`: An indexer for a local IOTA network.
-4. iota-node-`<OS>`-`<ARCHITECTURE>`: Run a local node.
-5. iota-tool-`<OS>`-`<ARCHITECTURE>`: Provides utilities for IOTA.
+1. iota: Main IOTA binary.
```

Comment:
Why remove this?

Reply:
@lucas-tortora, because this argument is not needed to install the binaries, some users might think they need to include it in the command to install the binaries. 

I mean this.

`<OS>`-`<ARCHITECTURE>`